### PR TITLE
Adding Lazydocker v0.23.3

### PR DESCRIPTION
**Pull request by eunos-1128**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by eunos-1128** [comment]
@conda-forge/help-go

**Comment by eunos-1128** [comment]
@conda-forge-admin, please ping conda-forge/

**Comment by eunos-1128** [comment]
@conda-forge/help-go, ready for review!

File: recipes/lazydocker/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+source:
+  url: https://github.com/{{ author }}/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 6ea52b69c3fb5cb371a01e55d1deeaf053b17fb240be069af577246c5169f4f1
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('go') }}
```
Comment (xhochy) [inline recipes/lazydocker/meta.yaml:18]:
```suggestion
    - {{ compiler('go-nocgo') }}
```

File: recipes/lazydocker/build.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -exuo pipefail
+
+export GO111MODULE=on
+export CGO_ENABLED=0
```
Comment (xhochy) [inline recipes/lazydocker/build.sh:6]:
```suggestion
```

File: recipes/lazydocker/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
+{% set name = "lazydocker" %}
+{% set version = "0.23.3" %}
+{% set author = "jesseduffield" %}
```
Comment (xhochy) [inline recipes/lazydocker/meta.yaml:3]: This is only used once, please don't use Jinja for it.
```suggestion
```

**Review by xhochy — CHANGES_REQUESTED** [review]
Please aggregate third-party licenses using `go-licenses`

File: recipes/lazydocker/meta.yaml (hunk 3 above)
Comment (eunos-1128) [reply recipes/lazydocker/meta.yaml]: This is used more than once.

https://github.com/conda-forge/staged-recipes/blob/baa18809804d28eb9fb82c69be76574e739fcabf/recipes/lazydocker/meta.yaml#L25-L33

**Review by xhochy — APPROVED** [review]

**Pull request merged**